projects/packages/jetpack-mu-wpcom/src/features/wpcom-site-menu/wpcom-site-menu.php

```diff
@@ -173,6 +176,9 @@ function add_all_sites_menu_to_masterbar( $wp_admin_bar ) {
 	if ( ! function_exists( 'wpcom_is_nav_redesign_enabled' ) || ! wpcom_is_nav_redesign_enabled() ) {
 		return;
 	}
+	if ( function_exists( 'is_agency_managed_site' ) && is_agency_managed_site() ) {
+		return;
+	}
```

Comment:
Hides "All Sites" menu in Masterbar.

<img src="https://private-user-images.githubusercontent.com/5634774/341844138-53daf306-b1c2-4cce-ad5c-b6d55811b101.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkwMDY1NjksIm5iZiI6MTcxOTAwNjI2OSwicGF0aCI6Ii81NjM0Nzc0LzM0MTg0NDEzOC01M2RhZjMwNi1iMWMyLTRjY2UtYWQ1Yy1iNmQ1NTgxMWIxMDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjFUMjE0NDI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzdhMDY2ZjZlYWUwNTNkM2Q5YmU5NDNlZGM2NzQ1MzQ0NzQ2ZDdkOTgxNDkxNGI2YTVhYjA0MzY0NDNmN2UzZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.hNh-_nn5-PmQmzsSZ-Vs456MFDOR9u24sfIrkK-9s-Y"/>